scripts/shfmt.bash

```diff
@@ -3,10 +3,8 @@
 set -euo pipefail
 
 # check .sh files
-# TODO(jthegedus): unlock this check later
-# TODO  shfmt --language-dialect posix --indent 2 --diff \
-# TODO  asdf.sh \
```

Comment:
Why remove `asdf.sh`?

Reply:
This is related to the thing I noted earlier:

> shfmt sometimes spuriously prints errors when encountering ZSH-specific syntax (./asdf.sh:34:16: parameter expansion requires a literal). Because of this, shfmt no longer runs on asdf.sh.